### ISSUE TITLE
[Bug] errors in documentation/examples

### ISSUE THREAD

**Issue by tsgit** [mariadb-operator/mariadb-operator#955]
**Documentation**
- [x ] I acknowledge that I have read the relevant [documentation](https://github.com/mariadb-operator/mariadb-operator/tree/main/docs).

**Describe the bug**

The slimmed down CRDs of v 0.34.0 define fewer properties than previously. In particular affinity and args are missing.
However the example here 
https://github.com/mariadb-operator/mariadb-operator/blob/main/examples/manifests/mariadb_galera_full.yaml#L93-L106
and the documentation here 
https://github.com/mariadb-operator/mariadb-operator/blob/main/docs/HA.md#pod-anti-affinity
still list affinity as a recognized property for initJob.

There may be other places that need to be updated. 

**Expected behaviour**

If the change was intentionl, the documentation and examples should be updated. If it is intentend for initJob to support those properties, the CRD should be amended. 

**Steps to reproduce the bug**

1. ...
2. ...
3. ...

**Debug information**
- Related object events:
```bash
kubectl get events --field-selector involvedObject.name=<mariadb-resource-name>
kubectl get events --field-selector involvedObject.name=<backup-resource-name>
kubectl get events --field-selector involvedObject.name=<restore-resource-name>
```
- `mariadb-operator` logs. Set the `--log-level` to `debug` if needed.

**Environment details**:
- Kubernetes version: [
Client Version: v1.31.1
Kustomize Version: v5.4.2
Server Version: v1.30.4
]
- Kubernetes distribution: [Vanilla]
- mariadb-operator version: [0.34.0]
- Galera data-plane version (init and agent): [0.34.0]
- Install method: [manifests]

**Additional context**

**Comment by mmontes11** [comment]
Hey there @tsgit ! Thanks for bringing this up.

In the case of the galera `initJob`, we didn't really saw an use case for the fields you mentioned:
- `args`: The init process is done based on environment variables already, we didn't see any useful use case for passing args as well.
- `affinity`: The init Job is the first Pod created during the provisioning process, so if no other Pods already exists, there is no point in creating anti-affinity policies.

Please, let me know if you have a valid use case for them, happy to add them back if there is a good reason. Otherwise I will update the docs in the [0.35.0](https://github.com/mariadb-operator/mariadb-operator/pull/950) version to be released this week.

Thanks!

**Comment by mmontes11** [comment]
Fixed by https://github.com/mariadb-operator/mariadb-operator/commit/b9db9557a7533280feb2a2d76947e3ab3afebce9. Will be released in [0.35.0](https://github.com/mariadb-operator/mariadb-operator/pull/950)

Closing! Please open another issue if you have an use case for this fields.

**Comment by tsgit** [comment]
Thanks for the quick response.
I don't have a use case for this. It tripped me up when updating to 0.34.0. My manifest followed the example.
Good riddance.
